templates/docs/examples/patterns/list-tree.html

```diff
@@ -56,14 +74,58 @@
   listTreeToggle.addEventListener('click', function(event) {
     event.preventDefault();
     var expand = this.getAttribute('aria-expanded') === 'true' ? false : true;
+
     toggleElement(listTreeToggle, expand);
   });
 }
 
+/**
+ * Adds or removes a class from an element.
+ * @param {HTMLElement} element - The element to add or remove the class from.
+ * @param {String} className - The class to add or remove.
+ * @param {Boolean} add - Whether to add or remove the class.
+ */
+function setClass(element, className, add) {
```

Comment:
Why are we adding an abstraction over a single line of code? Can't we use `element.classList.add/remove/toggle` directly?

Reply:
Removed the abstraction